package-lock.json

```diff
@@ -0,0 +1,285 @@
+{
+  "name": "wasm_simple_sample",
```

Comment:
This json file is generated by npm?

Reply:
Yes, correct. This file created automatically.